src/plugins/data/common/search/tabify/tabify_docs.ts

```diff
@@ -79,7 +80,7 @@ export const tabifyDocs = (
         const fieldName = field?.name || key;
         if (!columns.find((c) => c.id === fieldName)) {
           const fieldType = (field?.type as DatatableColumnType) || typeof value;
-          const formatter = field && index?.getFormatterForField(field);
+          const formatter = field && index?.getFormatterForField(field, formatParams);
```

Comment:
This is needed for https://github.com/elastic/kibana/pull/88303/files#diff-a86296d9011252ff17c1007bb3de66763c43af633f7daff1207151da34838e7fR225

But I am interested to know if there is a better way to set the timezone for field formatters in my other PR.